db/mongo_types.go

```diff
@@ -30,6 +30,10 @@ type OrgMembersCollection struct {
 	OrgMembers []OrgMember `json:"orgMembers" bson:"orgMembers"`
 }
 
+type OrgMembersGroupsCollection struct {
+	OrgParticipantsGroups []OrganizationMemberGroup `json:"organizationMemberGroups" bson:"organizationMemberGroups"`
```

Comment:
this should be `OrgMembersGroups`

Reply:
(anyway i see this struct is only used in String method which is currently unused AFAIU)